Automattic/wp-calypso
# Billing Management: deprecate the Upcoming Charges tab

#### Changes proposed in this Pull Request

* Remove the Upcoming Charges tab from the account level billing management pages and delete any newly unused code. I also removed the reference to this component from `https://wordpress.com/support/billing-history`.

#### Testing instructions

* Navigate to `/me/purchases` and verify that there is no "Upcoming Charges" tab in the nav bar.
* Manually load `/me/purchases/upcoming` and verify that you are redirected to `/me/purchases`.

![Screen Shot 2020-10-21 at 5 07 40 PM](https://user-images.githubusercontent.com/9310939/96793423-009ffc00-13c2-11eb-9047-95753463dd73.png)
